### DISCUSSION TITLE
Wann schaltet das Relais vom Hoymiles WR, wie sieht man es in den logs und wie miminiert man die Schaltvorgänge?

### DISCUSSION THREAD

**Discussion by gitisgreat2023** [hoylabs/OpenDTU-OnBattery#937] [Q&A] [answered]
Das Relais im Hoymiles Wechselrichter ist ggf limitierend für seinen Lebensdauer, ist halt ein Verschleißteil.
Jetzt fehlt mir den Überblick: wie kann ich sehen wir häufiger der geschaltet wird? Wie erkenne ich dass in logs? Welche Art von Einstellungen minimieren das benutzen des Relais?

Ich habe einen HM-1500, wenn jetzt die Sonne scheint und er nicht einspeist, steht da "Current Limit 30W | 2%". 
Wobei: Power DC steht zwar auf 7.8W aber Phase 1 zeigt 0.0 W und die Volt Zahl ändert sich. Ist das Relais jetzt dann geschaltet? Der Selbstverbrauch vom WR ist also dann etwa 8 Watt, richtig? Ja oder, sonst würde die Voltage sich ja nicht ändert, ist mit dem Netz verbunden.
Wenn dann doch der WR benötigt wird fährt er hoch und runter ohne dass das Relais geschaltet wird?

Zweit Frage:
Was bedeutet ID 124, shut down by remote control?

**Comment by spcqike** [comment, marked as answer]
soweit ich weiß, schaltet das Relais nach dem Zuschalten der Netzspannung, nach erfolgreicher Synchronisation mit dem Netz, zu. Es schaltet erst wieder ab, wenn entweder die Netzspannung wegfällt (Stromausfall, Sicherung, Stecker ziehen,...) oder wenn es ein Netzproblem (Überspannung, Unterspannung, Frequenzfehler,.....) gibt. 
Das stoppen und limitieren des Wechselrichters, auch wenn er solarbetrieben ist und die Sonne untergeht, führt nicht zu einem Abfall des Relais.

> Was bedeutet ID 124, shut down by remote control?

Dass der Wechselrichter das Signal bekommen hat, die Einspeisung zu stoppen. Von "Außen", also von der DTU.

**Reply by gitisgreat2023** [reply]
Alles klar hört sich logisch an. Die 8W sind dann standby Verbrauch?

**Reply by spcqike** [reply]
8W kenne ich von meinen so bisher nicht. Eher 2-3W. Aber das mag von Modell zu Modell variieren.

**Reply by gitisgreat2023** [reply]
Ich habe 2%, also 30W, HM-1500 also. Welche HM und welche min power hast du eingestellt?

**Reply by spcqike** [reply]
Einen HM-600. wenn ich ihn manuell stoppe hat er 0.7W pro String, 1.4W DC gesamt und eben 0W AC seitig. 

![IMG_5882](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/50098985/d470a0c1-de76-49d6-baac-c8352fcf8feb)

da ich ein reines PV System ohne Akku habe, ist meine min Leistung auf meine Grundlast (200W) gestellt. Er geht halt auch tagsüber normalerweise nicht aus.

**Comment by Feldsalat** [comment]
Ich habe zwei Inverter an meinem Akku. Der nicht produzierende HM-1200 zeigt dann auch erhöhte DC-Leistung an, wenn der andere Inverter gerade einspeist. So etwa 7W, wenn der andere gerade mit 200W einspeist. Normal sind ca. 1,3W pro MPPT im Standby.
